views/chat-view/src/components/ChatView/ChatView.tsx

```diff
@@ -65,19 +60,21 @@ export default function ChatView({
       editor.current?.clear();
 
       // @ts-ignore
-      const message = (await repo.create({ body: html })) as any;
+      const message = new Message(perspective, undefined, source);
+      message.body = html;
+      await message.save();
```

Comment:
It would make sense to use the new batch mode here for the message and the links below. Guess it would cut roundtrip time for messages showing up in half.